doc/source/whatsnew/v0.25.0.rst

```diff
@@ -159,28 +159,63 @@ Increased minimum versions for dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions.
-Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`24942`).
-If installed, we now require:
-
-+-----------------+-----------------+----------+
-| Package         | Minimum Version | Required |
-+=================+=================+==========+
-| beautifulsoup4  | 4.4.1           |          |
-+-----------------+-----------------+----------+
-| openpyxl        | 2.4.0           |          |
-+-----------------+-----------------+----------+
-| pymysql         | 0.7.9           |          |
-+-----------------+-----------------+----------+
-| pytz            | 2015.4          |          |
-+-----------------+-----------------+----------+
-| sqlalchemy      | 1.1.4           |          |
-+-----------------+-----------------+----------+
-| xlsxwriter      | 0.7.7           |          |
-+-----------------+-----------------+----------+
-| xlwt            | 1.0.0           |          |
-+-----------------+-----------------+----------+
-| pytest (dev)    | 4.0.2           |          |
-+-----------------+-----------------+----------+
+Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`24942`, :issue:`25554`, :issue:`25752`).
+
+Below is an overview of the minimal versions, as required resp. recommended. Optional libraries below the lowest tested version may still work, but are not considered supported.
```

Comment:
Not sure if that's what you had in mind in your [review](https://github.com/pandas-dev/pandas/pull/25752#issuecomment-476008611), but I think this is actually a good overview that:
* represents the way this is handled in practice
* is hard to find otherwise

Also, going through this exercise made me realise that by requiring the removal of the `travis-27` job, `pandas-gbq` was completely untested. I've ended up using `travis-36-locale` as a sort of compat-job for all the libraries that are not already covered by `azure-35-compat`.

Some other versions had no compat testing whatsoever (which does not fit with specifying minimum versions for them in `install.rst`), so I've added some.